packages/stream/package.json

```diff
@@ -25,12 +49,13 @@
     "jest": "29.3.1",
     "prettier": "2.8.1",
     "ts-jest": "29.0.3",
-    "typescript": "4.6.4"
+    "typescript": "^4.9.5"
   },
   "dependencies": {
```

Comment:
thinking about one package.json per repo. Will it be easier to manage cross-package dependencies sync in this case? 3rd parties like @solana/x, bn.js etc should be the same across all packages?